lib/generators/hyrax/hirmeos/templates/config/initializers/hyrax_hirmeos.rb

```diff
@@ -10,3 +10,8 @@
 Hyrax::Hirmeos::MetricsTracker.metrics_base_url = "https://metrics_base_url"
 Hyrax::Hirmeos::MetricsTracker.translation_base_url = "https://translations_base_url"
 Hyrax::Hirmeos::MetricsTracker.secret = ENV['HIRMEOS_TRANSLATOR_KEY']
+
+Hyrax.config.callback.set(:after_create_fileset) do |file_set, user|
```

Comment:
Do you think you could add a test which triggers this callback and check that both jobs get enqueued only once?  I think you could call the callback just like in the FileSetActor: `Hyrax.config.callback.run(:after_create_fileset, file_set, user)`